src/crepe/api/AI.cpp

```diff
@@ -0,0 +1,87 @@
+#include <stdexcept>
+
+#include "AI.h"
+
+namespace crepe {
+
+AI::AI(game_object_id_t id, float max_force) : Component(id), max_force(max_force) {}
+
+void AI::make_circle_path(float radius, const vec2 & center, float start_angle,
+						  bool clockwise) {
+	if (radius <= 0) {
+		throw std::runtime_error("Radius must be greater than 0");
+	}
+
+	// The step size is determined by the radius (step size is in radians)
+	float step = 400.0f / radius;
+	// Force at least 16 steps (in case of a small radius)
+	if (step > 2 * M_PI / 16) {
```

Comment:
veel magic waardes en dezelfde

Reply:
Alleen de 400 is een magic waarde, toch? 2Pi lijkt mij vanzelfsprekend (radialen) en de 16 ligt ik erboven toe.

Reply:
je hebt heel vaak 16 als MIN_STEP kun je dit een constexpr maken?